pages/platform/kubernetes-k8s/getting-source-ip-behind-loadbalancer/guide.en.gb.md

```diff
@@ -0,0 +1,309 @@
+---
+title: 'Getting the source IP behind the LoadBalancer'
+slug: getting-source-ip-behind-loadbalancer
+excerpt: 'Find out how to get the source IP behind the LoadBalancer on OVH Managed Kubernetes'
+section: Tutorials
+---
+
+<style>
+ pre {
+     font-size: 14px;
+ }
+ pre.console {
+   background-color: #300A24; 
+   color: #ccc;
+   font-family: monospace;
+   padding: 5px;
+   margin-bottom: 5px;
+ }
+ pre.console code {
+   border: solid 0px transparent;
+   font-family: monospace !important;
+   font-size: 0.75em;
+   color: #ccc;
+ }
+ .small {
+     font-size: 0.75em;
+ }
+</style>
+
+**Last updated 20<sup>th</sup> August, 2019.**
+
+
+## Before you begin
+
+This tutorial presupposes that you already have a working OVH Managed Kubernetes cluster, and you have deployed there an application using the OVH Managed Kubernetes LoadBalancer. If you want to know more on those topics, please look at the [using the OVH Managed Kubernetes LoadBalancer](../using-lb/) documentation.
+
+
+## The problem
+
+When you deploy your HTTP services in `NodePort` mode, you directly recover the request's `Remote Address` from the server (for example using `$_SERVER['REMOTE_ADDR']` on PHP or `$ENV{'REMOTE_ADDR'}` in Perl). This address (usually in `IP:port` format) corresponds to the original requestor **or** the last proxy between them and your cluster.
+
+When deploying the services in `LoadBalancer` mode, things are a bit different, our Load Balancer acts like a proxy, and the `Remote Address` will give you the IP address of the Load Balancer. How can you get the source IP of the request in this case?
+
+This tutorial describe how to deploy a `LoadBalancer` service on OVH Managed Kubernetes and preserve the source IP.
+
+
+## Getting the request's source IP behind the LoadBalancer
+
+The easiest way to deploy services behind the Load Balancer while keeping the source IP is to place your services under an `Ingress`, itself behind the `LoadBalancer`. 
+
+The `Ingress` is exposed to the outside of the cluster either via  `LoadBalancer`, and it routes incoming traffic to your services according to configured rules. And additional advantage of this setting is the cost: you can have lots of services behind a single `LoadBalancer`.
+
+In this tutorial we are using the most basic Ingress Controller: [NGINX Ingress Controller](https://github.com/kubernetes/ingress-nginx){.external}, where an NGINX server take the role of reverse proxy.
+
+
+### 1. Installing the NGINX Ingress Controller
+
+The official way to install the **NGINX Ingress Controller* is using a mandatory manifest file:
```

Comment:
I think it miss a star at the end of 'NGINX Ingress Controller'

Reply:
Fixing, thanks!